web/sites/default/config/core.entity_view_display.taxonomy_term.council_committee.default.yml

```diff
@@ -42,8 +47,78 @@ third_party_settings:
               label_display: '0'
               context_mapping:
                 entity: layout_builder.entity
+            weight: 6
+            additional: {  }
+          dc1e5ad1-da35-49ca-abf7-6f8e4c70d633:
```

Comment:
Could we move the committee description block to the top, above all the committee members?

Reply:
Moved the Committee description in layout builder above committee members.